# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mathieu/xcdroast/10_dvd_atip.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ru xcdroast-0.98alpha14-/src/io.c xcdroast-0.98alpha14/src/io.c
--- xcdroast-0.98alpha14-/src/io.c	2003-05-29 15:14:42.000000000 +0200
+++ xcdroast-0.98alpha14/src/io.c	2003-06-15 22:42:15.000000000 +0200
@@ -5507,7 +5507,11 @@
 	}
 
 	/* build command line */
-	get_wrap_path_cdrecord(tmp3);
+	if (curset.isProDVD) {
+        	get_wrap_path("CDRECORDPRODVD", tmp3);
+	} else {
+        	get_wrap_path("CDRECORD", tmp3);
+	}
 	g_snprintf(line,MAXLINE,"%s %s -atip 2>&1",
 		tmp3,tmp);
 
@@ -5518,6 +5522,11 @@
                 g_error("popen error\n");
         }
 
+	/* syntax for dvd-atip is different */
+	if (is_dvdwriter(curset.writer_devnr)) {
+		found_atip = 1;
+	}
+
 	errcount = 0;
         for (;;) {
                 if (fgets(line,MAXLINE,fpin) == NULL) 
